Sources/TokamakCore/Modifiers/ViewModifier.swift

```diff
@@ -0,0 +1,34 @@
+// Copyright 2020 Tokamak contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+public protocol ViewModifier {
+    typealias Content = _ViewModifier_Content<Self>
+    associatedtype Body : View
```

Comment:
I've just added [the "Coding Style" section](https://github.com/swiftwasm/Tokamak#coding-style) to README.md, sorry about the delay. Would you be able to install SwiftFormat if you don't have it yet and run `swiftformat .` in your branch to make the whitespaces here more consistent with the rest of the codebase? Thanks!